app/components/onboarding_email_form/onboarding_email_form.html.erb

```diff
@@ -31,6 +31,18 @@
         <%= c 'checkbox', **field.input_defaults, required: true %>
       <% end %>
 
+      <% if Setting.ask_for_additional_consent && Setting.additional_consent_defined %>
+        <%= c('field',
```

Comment:
I think we should extract the consent checkboxes into a separate component, although we may do this after this PR is merged.

Reply:
100% agree on this one, will put it on the follow-up issue